lib/tasks/factory_girl.rake

```diff
@@ -0,0 +1,15 @@
+namespace :factory_girl do
```

Comment:
@Kevin-Wei I like where your head's at with this. Let me mull over the best way to work this in.